src/main/java/com/rabbitmq/jms/client/RMQConnection.java

```diff
@@ -82,6 +84,13 @@
     private volatile boolean canSetClientID = true;
 
     /**
+     * QoS setting for channels created by this connection.
+     *
+     * @see com.rabbitmq.client.Channel#basicQos(int)
+     */
+    private final int channelsQos;
```

Comment:
Does this value have to be `final`? QoS changes on the fly are not very common but possible in AMQP 0-9-1.